CHANGELOG.md

```diff
@@ -7,6 +7,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+### Added
+
+- Add optional parameter `enableWebAudio: boolean` in the config of meeting manager to decide whether to enable web audio in the device controller.
+- Extend the ability to accept device and audioTransformDevice as audio input in meeting manager.
+- Add a new provider `VoiceFocusProvider` to initial the workflow of voice focus, and transform a normal device to voice focus device.
+- Add a new UI component `AudioInputVFControl` to provider voice focus option for users.
```

Comment:
nit: provider -> provide, for users -> to users?

Reply:
updated